go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/longhorn/backupstore v0.0.0-20240720163059-56c90cd23634
 	github.com/longhorn/go-common-libs v0.0.0-20240727052403-b543ea3732fe
 	github.com/longhorn/go-spdk-helper v0.0.0-20240727080828-a55a2982815a
-	github.com/longhorn/longhorn-engine v1.7.0-rc2
-	github.com/longhorn/longhorn-spdk-engine v0.0.0-20240727114504-89a392e62633
+	github.com/longhorn/longhorn-engine v1.7.0-rc1
```

Comment:
Why this was returned back to the previous version/digest?

Reply:
Weird. Updating again.